src/components/pages/CurriculumInfo/helpers/curriculumMetadata.ts

```diff
@@ -0,0 +1,40 @@
+import { capitalize } from "lodash";
+
+export const buildCurriculumMetadata = (props: {
+  metadataType: "title" | "description";
+  tab: "units" | "overview";
+  keyStagesData: string;
+  subjectSlug: string;
+  examboardSlug?: string | null;
+}) => {
+  if (props.tab === "overview" && props.metadataType === "title") {
+    const examboard =
+      props.examboardSlug && props.examboardSlug !== "aqa"
```

Comment:
Could this be something like the below? 
```suggestion
      props.examboardSlug && props.examboardSlug === "aqa" ? "AQA" : 
      props.examboardSlug ? ` ${capitalize(props.examboardSlug)}` : ""
```